service/src/main/java/org/databiosphere/workspacedataservice/service/ImportService.java

```diff
@@ -95,6 +96,12 @@ public GenericJobServerModel createImport(
       arguments.put(ARG_TOKEN, petToken);
       arguments.put(ARG_URL, importRequest.getUrl().toString());
       arguments.put(ARG_COLLECTION, collectionId.toString());
+      // try to retrieve the tdrSyncPermissions option if available
+      arguments.put(
+          ARG_PERMISSION_OPTION,
+          importRequest.getOptions().containsKey("tdrSyncPermissions")
+              ? (boolean) importRequest.getOptions().get("tdrSyncPermissions")
```

Comment:
```suggestion
              ? Boolean.parseBoolean(importRequest.getOptions().get("tdrSyncPermissions"))
```

I suspect you need to do more than cast here

Reply:
I believe you, but for my learnings sake, why? The map is <string, object>, so if you have an object that's just `true`, wouldn't that (boolean) cast be sufficient?